src/ansys/fluent/core/workflow.py

```diff
@@ -911,7 +911,9 @@ def __getattr__(self, attr):
         )  # or self._task_with_cmd_matching_help_string(attr)
         if obj:
             return obj
-        return self._task_objects.get(attr, None)
+        if attr in self._task_objects:
+            return self._task_objects[attr]
+        super().__getattribute__(attr)
```

Comment:
@mkundu1 Is this line meant to return the value returned by the call?

Reply:
@seanpearsonuk, yes. This provides an attribute error for non-existing attr instead of returning None as well.

Reply:
Yes, this should return the value @prmukherj

Reply:
Done.